user_sync/config.py

```diff
@@ -186,10 +188,14 @@ def load_invocation_options(self):
                 raise AssertionException('You cannot specify --adobe-only-user-list when using "push" strategy')
             users_spec = None
             stray_list_input_path = self.args['adobe_only_user_list']
-        elif options['users'] and options['adobe_only_user_list']:
-            raise AssertionException('You cannot configure both a default "users" option (%s) '
-                                     'and a default "adobe-only-user-list" option (%s)' %
-                                     (' '.join(options['users']), options['adobe_only_user_list']))
+
+        # Check both that options['users'] exists, AND that it is not the default value before checking against
+        # the adobe_only_user_list parameter -  otherwise, defaults cause exception to be thrown unconditionally when
+        # the user list option is specified
+        elif (options['users'] and options['users'] != self.invocation_defaults['users']) and options['adobe_only_user_list']:
+                raise AssertionException('You cannot configure both a default "users" option (%s) '
```

Comment:
@janssenda-adobe Will you please fix the indentation on this raise statement so it's just one level indented from the elif block?  It doesn't seem to be causing any issues, but a linter would probably complain about it, and we want the diff to render accurately.

Reply:
@adorton-adobe Thanks! Fixed that